google-cloud-firestore/src/main/java/com/google/cloud/firestore/UpdateBuilder.java

```diff
@@ -535,13 +527,11 @@ public boolean allowTransform() {
     Mutation mutation = addMutation();
```

Comment:
I think we can probably remove the `Mutation` type and replace it with the Proto type. What do you think?